src/components/Categorization.tsx

```diff
@@ -0,0 +1,11 @@
+import starIcon from "../assets/star.svg";
```

Comment:
Extracting the rating component and passing the number of stars you wish to show is a great idea. This is a good example of generalization and reusability. Good job